src/extensions/web-vitals/index.ts

```diff
@@ -200,7 +200,10 @@ export class WebVitalsAutocapture {
         if (!isUndefined(posthogExtensions)) {
             ;({ onLCP, onCLS, onFCP, onINP } = posthogExtensions.postHogWebVitalsCallbacks)
         }
-
+        else if (assignableWindow.postHogWebVitalsCallbacks) {
```

Comment:
hey @brunomguimaraes this shouldn't be necessary, what's the problem this solves?

Reply:
Ugh, dang... It's because we load the latest web vitals extension into older clients. What a pain 😭 

AFK right now will look at this properly when back

Reply:
:)

I should have caught it... have opened https://github.com/PostHog/posthog-js/pull/1407 to try and work towards making this all safer